node/src/actor_system_factory.rs

```diff
@@ -334,7 +334,12 @@ impl ActorSystemFactoryToolsReal {
     }
 
     fn handle_automap_error(prefix: &str, error: AutomapError) {
-        exit_process(1, &format!("Automap failure: {}{:?}", prefix, error));
+        let msg = format!("Automap failure: {}{:?}", prefix, error);
+        if error.should_crash() {
+            exit_process(1, &format!("Automap failure: {}{:?}", prefix, error));
+        } else {
+            todo!("What do we do if we don't crash? ({})", msg)
```

Comment:
Well, I don't know the answer, honestly, but the clear thing is that we don't want to panic on this `todo!()`. So please return to it and give it more thoughts. 

My suggestion: I'd act accordingly to the place of use of this function. In overall, though, it would be attempts to find a way how to log and let it run on or evaporate. I don't know if there are chances to find a similar solution.

Reply:
I decided to log and return.